### PR TITLE
CLDR-15297 generate coverage charts for v41

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15297

- [ ] This PR completes the ticket.

### CODE REVIEW

**Review by pedberg-icu — APPROVED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/Level.java [hunk 1]
```diff
@@ -12,14 +12,14 @@
  *
  */
 public enum Level {
-    UNDETERMINED(0, "none", 0),
-    CORE(10, "G4", 100),
-    BASIC(40, "G3", 80),
-    MODERATE(60, "G2", 70),
-    MODERN(80, "G1", 50),
-    COMPREHENSIVE(100, "G0", 2);
+    UNDETERMINED(0, "none", 0, "�"),
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/Level.java:15]: intentionally U+FFFD? Would something else be better?

**Review by srl295 — APPROVED** [review]
lgtm with comment

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/Level.java (hunk 1 above)
Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/Level.java:15]: It should never show up, so that's just a visual reminder!

**Comment by macchiati** [comment]
Had to squash manually; can one of you rereview?

**Pull request merged**